node/forest_libp2p/tests/decode_test.rs

```diff
@@ -38,17 +43,19 @@ fn convert_single_tipset_bundle() {
     .into_result()
     .unwrap();
 
-    assert_eq!(res, [FullTipset::new(vec![])]);
+    assert_eq!(res, [FullTipset::new(vec![block]).unwrap()]);
 }
 
 #[test]
 fn tipset_bundle_to_full_tipset() {
     let h0 = BlockHeader::builder()
         .weight(BigUint::from(1u32))
+        .miner_address(Address::new_id(0))
         .build()
         .unwrap();
     let h1 = BlockHeader::builder()
-        .weight(BigUint::from(2u32))
+        .weight(BigUint::from(1u32))
+        .miner_address(Address::new_id(1))
```

Comment:
@austinabell Let me know if this is the right way to handle this, I had to make the weights equal and the miners distinct in order to pass `FullTipset::new`'s validation

Reply:
Yup! that's intended, I'll quote the spec:

```
A valid tipset contains a non-empty collection of blocks that have distinct miners and all specify identical:

Epoch
Parents
ParentWeight
StateRoot
ReceiptsRoot
```